services/workflows-service/src/workflow/cron/ongoing-monitoring.cron.ts

```diff
@@ -71,7 +71,7 @@ export class OngoingMonitoringCron {
             const lastReceivedReport = await this.findLastBusinessReport(business, projectIds);
 
             if (!lastReceivedReport) {
-              this.logger.error(`No initial report found for business: ${business.id}`);
+              this.logger.debug(`No initial report found for business: ${business.id}`);
```

Comment:
print an info instead

Reply:
it's not an info, this would bombard our logs.